src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/FormFileModelBinder.cs

```diff
@@ -33,69 +33,115 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
             // using Task.FromResult. If you need to make changes of this nature, profile
             // allocations afterwards and look for Task<ModelBindingResult>.
```

Comment:
Didn't update all of these but did touch the couple in files already part of this PR. Please confirm new wording (there's still _an_ optimization) in next iteration.